rules_ruby_gapic/ruby/bundle_install.bzl

```diff
@@ -0,0 +1,123 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Creates a workspace that installs bundled gems for the bundle execution later
+"""
+load(
+  ":private/utils.bzl",
+  _execute_log_action = "execute_log_action",
+)
+
+def _bundle_install(repo_ctx):
+  # # gemfile
+  # orig_gemfile_path = repo_ctx.path(repo_ctx.attr.gemfile)
+  # gemfile_path = repo_ctx.path("./export/gapic-generator-cloud/Gemfile")
+  # repo_ctx.file(gemfile_path, "") # this create the folder structure like mkdir -p
+  # _execute_log_action(repo_ctx, "copy_gemfile.log", 
+  #   ["cp", "%s" % orig_gemfile_path, "%s" % gemfile_path]
+  # )
```

Comment:
What's going on with these commented out lines?

Reply:
nice catch, the copying has been unified and these are no longer necessary